src/layout/Projects.astro

```diff
@@ -36,6 +37,7 @@ const {
 		<style is:inline set:html={resetStyles}></style>
 		<link rel="icon" type="image/x-icon" href="favicon.ico">
 		<SlideUtils/>
+		<Debounce/>
```

Comment:
@Oluwa-nifemi I think these two guys should be Utils functions that are imported on a need-to basis.

Rule of thumb, if it's going to be used multiple times and the code doesn't have HTML and CSS components, then it should most likely be in the utils directory and imported into the <script> that needs it. Do you get?

I'll merge these, but kindly not in a next PR

Reply:
Yeah I understand